js/index.js

```diff
@@ -1,5 +1,5 @@
 mapboxgl.accessToken =
-    "";
+    "pk.eyJ1IjoiY2FsbHVtZGVubmlzaWUiLCJhIjoiY2xrMWY2a2U0MDFzYTNmbHNreTB4ejFiYiJ9.nxsaRwn_FmKun20KrOdjNw";
```

Comment:
[issue blocking] The API key is being added by the changes. You might need to remove it before moving forward with the pull request.